### PR TITLE
Various updates

### PR DESCRIPTION
**Pull request by jmhauck**
#124

### CODE REVIEW

File: src/assets/t9n/public-notification/resources.json [hunk 1]
```diff
@@ -25,5 +25,8 @@
     "selectSketchTip": "Draw shapes to select features from the addressee layer",
     "optionalSearchDistance": "and optionally add search distance.",
     "listHasSetsTip": "For existing notification lists optionally edit or remove each list. Also, refine across all selection sets or export to PDF or CSV.",
-    "uniqueCout": "Unique Count: {{n}}"
+    "uniqueCout": "Unique Count: {{n}}",
```
Comment (MikeTschudi) [inline src/assets/t9n/public-notification/resources.json:28]: "uniqueCount" would be clearer here for our translators

**Review by MikeTschudi — DISMISSED** [review]
Is it OK to send it out in this release without translations for the new strings?

**Comment by jmhauck** [comment]
@MikeTschudi Kelly said as long as we got it in by tomorrow morning all would be good

File: src/assets/t9n/public-notification/resources.json (hunk 1 above)
Comment (jmhauck) [reply src/assets/t9n/public-notification/resources.json:28]: @MikeTschudi Good catch, I will update this after we get the translations back from the IA team

**Comment by jmhauck** [comment]
And sorry...I spaced that this was still open before I started pushing more commits

**Review by MikeTschudi — APPROVED** [review]

**Pull request merged**